editor.planx.uk/src/pages/FlowEditor/components/Team/TeamMembers.tsx

```diff
@@ -1,19 +1,158 @@
+    return roleLabels[role] || role;
+  };
+
+  const renderMemberTable = (members: TeamMember[]) => {
+    if (members.length === 0) {
+      return (
+        <Table>
+          <TableHead>
+            <TableRow>
+              <TableCell>
+                <strong>No members found</strong>
+              </TableCell>
+            </TableRow>
+          </TableHead>
+        </Table>
+      );
+    }
+    return (
+      <TableContainer>
+        <Table>
+          <TableHead>
+            <StyledTableRow>
+              <TableCell sx={{ width: 300 }}>
+                <strong>User</strong>
+              </TableCell>
+              <TableCell sx={{ width: 200 }}>
+                <strong>Role</strong>
+              </TableCell>
+              <TableCell>
+                <strong>Email</strong>
+              </TableCell>
+            </StyledTableRow>
+          </TableHead>
+          <TableBody>
+            {members.map((member) => (
+              <StyledTableRow key={member.id}>
+                <TableCell
+                  sx={{
+                    display: "flex",
+                    flexDirection: "row",
+                    alignItems: "center",
+                  }}
+                >
+                  <StyledAvatar>
+                    {member.firstName[0]}
+                    {member.lastName[0]}
+                  </StyledAvatar>
+                  {member.firstName} {member.lastName}
+                </TableCell>
+                <TableCell>
+                  <Chip
+                    label={getRoleLabel(member.role)}
+                    size="small"
+                    sx={{ background: "#ddd" }}
+                  />
+                </TableCell>
+                <TableCell>{member.email}</TableCell>
+              </StyledTableRow>
+            ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+    );
+  };
+
   return (
-    <Box component="pre" sx={{ fontSize: 12 }}>
-      {JSON.stringify(teamMembersByRole, null, 4)}
-    </Box>
+    <Container maxWidth="contentWrap">
+      <Box py={7}>
+        <EditorRow>
+          <Typography variant="h2" component="h3" gutterBottom>
+            Team editors
+          </Typography>
+          <Typography variant="body1">
+            Editors have access to edit your flows and services.
+          </Typography>
+          {renderMemberTable(otherRoles)}
+        </EditorRow>
+        <EditorRow>
+          <Typography variant="h2" component="h3" gutterBottom>
+            Admins
+          </Typography>
+          <Typography variant="body1">
+            Admins have editor access across all teams.
+          </Typography>
+          {renderMemberTable(platformAdmins)}
+        </EditorRow>
+        <EditorRow>
+          <Typography variant="h2" component="h3" gutterBottom>
+            Archived team editors
+          </Typography>
+          <Typography variant="body1">
+            Past members who no longer have access to edit your team.
```

Comment:
nit: maybe "Past team members who no longer have access to the Editor, but may be part of the edit history of your services" provides a little more context?

Reply:
Makes a lot more sense written this way, with continuity of referring to "services" 👍